### PR TITLE
add WeAre-photo album of AR

### PR DESCRIPTION
**Pull request by SherlockQi**
1. [x] Project URL:https://itunes.apple.com/app/id1234567890
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Screenshot included 
5. [x] Avoid iOS or open-source in description as it is assumed
6. [x] Use this commit title format if applicable: Add app-name by @github_heikki
7. [x] Use approved format for your entry

### CODE REVIEW

**Comment by SherlockQi** [comment]
sorry.I don't understand.

File: contents.json [hunk 1]
```diff
@@ -17151,6 +17151,23 @@
...
+      "source": "https://github.com/SherlockQi/HeavenMemoirs",
+      "homepage": "https://github.com/SherlockQi",
+      "itunes": "https://itunes.apple.com/app/id1234567890",		
+      "lang": "zho",     
+      "stars": 427,     
+      "tags": [
+        "swift",
+        "arkit"
+       ],
+      "screenshots": ["https://github.com/SherlockQi/HKNote/blob/master/image/WeAre.gif"],
```
Comment (vpeschenkov) [inline contents.json]: @SherlockQi there should be a direct link to an image/gif, please, change the link.

File: contents.json [hunk 2]
```diff
@@ -17151,6 +17151,23 @@
...
       "suggested_by": "@dkhamsing"
-    }
+    },
+    {
+      "title": "WeAre",
+      "category-ids": ["photo"],
+      "description": "photo album of AR",
+      "source": "https://github.com/SherlockQi/HeavenMemoirs",
+      "homepage": "https://github.com/SherlockQi",
+      "itunes": "https://itunes.apple.com/app/id1234567890",
```
Comment (vpeschenkov) [inline contents.json]: @SherlockQi replace the iTunes link by this `https://itunes.apple.com/app/weare/id1304227680` link, please.

**Comment by vpeschenkov** [comment]
Hello @SherlockQi, thank you for contributing, could you please fix the review issues? 

> sorry.I don't understand.

Do not worry we will take care of other issues by ourselves 🙂

File: contents.json [hunk 3]
```diff
@@ -17151,6 +17151,24 @@
...
+    },
+    {
+      "title": "WeAre",
+      "category-ids": [
+        "photo"
+      ],
+      "description": "photo album of AR",
+      "source": "https://github.com/SherlockQi/HeavenMemoirs",
+      "homepage": "https://github.com/SherlockQi",
+      "itunes": "https://itunes.apple.com/app/id1234567890",
```
Comment (vpeschenkov) [inline contents.json]: @SherlockQi https://itunes.apple.com/app/id1234567890 doesn't work for me. Could you change this link to a working link?

File: contents.json [hunk 4]
```diff
@@ -17151,6 +17151,24 @@
...
+      "source": "https://github.com/SherlockQi/HeavenMemoirs",
+      "homepage": "https://github.com/SherlockQi",
+      "itunes": "https://itunes.apple.com/app/id1234567890",		
+      "lang": "zho",     
+      "stars": 427,     
+      "tags": [
+        "swift",
+        "arkit"
+       ],
+      "date_added": "Dec 5 2018",
```
Comment (vpeschenkov) [inline contents.json]: @SherlockQi we need at least one screenshot of this app.

**Comment by vpeschenkov** [comment]
@SherlockQi thank you!

**Pull request merged**